### PR TITLE
fix: Firefox miss bottom line when use sorter

### LINKED ISSUE #16131: table加上排序后会 遮挡下边框 怎么处理 Firefox和edge浏览器
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/table-cn/#components-table-demo-ajax](https://ant.design/components/table-cn/#components-table-demo-ajax)

### Steps to reproduce
https://ant.design/components/table-cn/#components-table-demo-ajax

### What is expected?
修复

### What is actually happening?
修复

| Environment | Info |
|---|---|
| antd | 3.16.3 |
| React | 最新 |
| System | win10 |
| Browser | 66.0.3 |

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

close #16131 

### 💡 Solution

Set background color of tr as workaround. But still will facing Chrome hover blink.

### 📝 Changelog

Fix Firefox miss bottom line when use sorter.

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/style/index.less [hunk 1]
```diff
@@ -32,169 +32,174 @@
...
-      top: 0;
-      right: 0;
-      width: 28px;
-      height: 100%;
-      color: @table-header-icon-color;
-      font-size: @font-size-sm;
-      text-align: center;
-      cursor: pointer;
-      transition: all 0.3s;
+    & > th {
```
Comment (afc163) [inline components/table/style/index.less]: & 是多余的

**Comment by afc163** [comment]
改了好多，具体做法是？

**Comment by zombieJ** [comment]
> 改了好多，具体做法是？

1. 设置 `.ant-thead > tr > th` 默认背景为 `transparent`
2. 设置 `.ant-thead > tr` 背景为 `@table-header-bg`

因为抽了一层，看起来改了很多。

**Review by ztplz — APPROVED** [review]

**Comment by afc163** [comment]
图片

hover 的时候，其他列变白了。

File: components/table/style/index.less [hunk 2]
```diff
@@ -230,10 +235,6 @@
     }
   }
 
-  &-thead > tr:hover {
-    background: none;
-  }
-
```
Comment (zombieJ) [inline components/table/style/index.less]: 这边移除了 标题 hover 的样式，看起来没什么影响。帮忙 double confirm 一下：
@afc163 https://github.com/ant-design/ant-design/commit/27a773031cad8397b40429fb602e074926a31147

File: components/table/style/index.less [hunk 3]
```diff
@@ -32,169 +32,174 @@
...
 
-  &-thead > tr > th {
-    color: @table-header-color;
-    font-weight: 500;
-    text-align: left;
+  &-thead > tr {
+    // For Chrome extra space:
+    // - https://github.com/ant-design/ant-design/issues/14926
+    // - https://github.com/ant-design/ant-design/issues/16131
     background: @table-header-bg;
```
Comment (afc163) [inline components/table/style/index.less]: cause https://github.com/ant-design/ant-design/issues/16933

**Pull request merged**